# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/tests/browser/browser.ini
# Commit: 3f6deedb2ffa
# Full Hash: 3f6deedb2ffae2a52c130f68cc7c0bda882bedfd
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-02-08 17:32:27
# Regressor Bug: 1746104
# File Overlap Count: 1
# Description:
#   Bug 1746104 - part 7: Make `eQuerySelectedText` event and `ContentEventHandler` work with no selection ranges r=m_kato
#   
#   Finally, making `WidgetQueryContentEvent::Succeeded` return `true` when there
#   is no selection when its message is `eQuerySelectedText`, and making
#   `ContentEventHandler::OnQuerySelectedText` set the no selection range data even
# ==============================================================================

diff -r 989ad9507a0c -r 3f6deedb2ffa widget/tests/browser/browser.ini
--- a/widget/tests/browser/browser.ini	Mon Feb 07 22:33:41 2022 +0000
+++ b/widget/tests/browser/browser.ini	Mon Feb 07 22:33:41 2022 +0000
@@ -1,5 +1,7 @@
 [browser_test_clipboardcache.js]
 skip-if = os == 'android' || (os == 'linux' && ccov) || tsan # Bug 1613516, the test consistently timeouts on Linux coverage builds.
+[browser_test_ContentCache.js]
+skip-if = os == 'android'
 [browser_test_swipe_gesture.js]
 run-if = (os == 'mac' || os == 'win')
 support-files =